app/views/static_error_pages/_error_page.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :head do %>
```

Comment:
I think the emergency banner also needs to be added here.

Reply:
Ah, no - that was specifically removed, but I do need to mention that in the commit somehow (https://gds.slack.com/archives/C06SVL1GC2Y/p1732628784914559)

Reply:
I've added the reasoning into the commit. I didn't bother putting the slack link in because it'll expire, but I'll add a transcript of it into the trello card so there's at least some way of getting to it.